app/ios/OpenPassport/Info.plist

```diff
@@ -39,6 +39,8 @@
 	<false />
 	<key>NSAllowsLocalNetworking</key>
```

Comment:
btw do we need this? when i open app for first time it asked to connect to local area devices but i dont know why